### ISSUE TITLE
Hianime extension not working

### ISSUE THREAD

**Issue by spidy2356** [Dark25/aniyomi-extensions#154]
### Source information

HiAnime 14.47

### Source language

English

### Steps to reproduce

1. Open any anime from hianime (for eg. One piece)
2. It says video list empty

### Expected behavior

It should play video

### Actual behavior

Any resources from hianime says "video list empty"
![Screenshot_20241213-223904](https://github.com/user-attachments/assets/5e5618c0-2a7d-4147-9ce4-774b410d553b)

### Aniyomi version

Version 0.16.4.3-8019 xyz.jmir.tachiyomi.mi.debug

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.2.4](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/docs/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by alphastark** [comment]
I'm having the exact same issue with the HiAnime extension

**Comment by spidy2356** [comment]
> I'm having the exact same issue with the HiAnime extension

Yes, some anime working but some r not...for eg one piece not working but death note working.

Someone from discord of kohi den server says me, 

"the sites video providers, megacloud have made changes recently and the extension cannot access it, something to do with how the extension sends/receives the video data.

Hianime site itself also changed their encryption keys.

Currently, there is nothing that can be done until:
- someone gets those new keys
- figure out how megaclouds new changes actually work. "

**Comment by Dark25** [comment]
fixed

**Comment by alphastark** [comment]
> fixed

Thank you!!!
